Website/DesktopModules/Hotcakes/Core/Controllers/CartController.cs

```diff
@@ -342,6 +343,34 @@ public ActionResult Index()
             return View(model);
         }
 
+        // GET: /MiniCart/
+        [NonCacheableResponseFilter]
+        public ActionResult MiniCart()
+        {
+            var model = new MiniCartViewModel
+            {
+                TotalQuantity = (CurrentCart != null && CurrentCart.Items != null) ? CurrentCart.Items.Count : 0
+            };
+
+            return View(model);
+        }
+
+        // POST: /MiniCartItems
+        [ActionName("MiniCartItems")]
+        [HccHttpPost]
+        public JsonResult GetMiniCartItems()
```

Comment:
@ismaelTartola Should we have any error handling here?  If something goes wrong, nothing is logged, and the UI doesn't know what to do.

Reply:
@ismaelTartola In the very least, there should be some sort of `try ... catch` in the endpoint itself.  If something goes wrong, the error is logged to the Log4NET log file.  There should also be some sort of generic error message in the UI.  Something stating, "An unknown error has occurred. If this continues to happen, please contact the store directly."  

You might want to use the built-in UI messaging.  https://uxguide.dnnsoftware.com/UIPatterns/ModuleMessages.html

Reply:
@WillStrohl Like I commented in slack today I haven't been able to find any error handling or logging feature implementation, so I have added a custom implementation of the features(logging and error handling) to the actions involved in the minicart skin object functionalities. But that is so rare, I have even take the liberty to explore the many of the other hotcackes modules controllers and I'm haven't found a single implementation for logging or error handling anywhere. That concern me, I have to point that I really think we are missing something. I my mind I just can't think that this the way this solution is implemented.